examples/backbone/js/views/app-view.js

```diff
@@ -70,7 +71,7 @@ var app = app || {};
 				this.$footer.hide();
 			}
 
-			this.allCheckbox.checked = !remaining;
+			this.allCheckbox.checked = size && !remaining;
```

Comment:
I am not able to ever hit this new check.. can you drop this change and open a PR with only the debounce change and we can get this in. thanks again @elie29